pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.580.1</version>
+        <version>1.580.3</version>
```

Comment:
:bug: It forces the plugin to be incompatible with user installations on 1.580.1